### PR TITLE
Waypoints around KLAX (and the first fix)

### PR DESCRIPTION
**Pull request by liamkirky**
first fix yay!

### CODE REVIEW

File: Fixes.json [hunk 1]
```diff
@@ -326635,6 +326640,9 @@
     "Longitude": -68.21154
   },
   {
+    "Latitude":
```
Comment (Adrien-Z) [inline Fixes.json]: What is that?

File: Fixes.json [hunk 2]
```diff
@@ -430239,6 +430252,8 @@
     "Name": "POMOK",
     "Longitude": 121.116669
   },
+  {
```
Comment (Adrien-Z) [inline Fixes.json]: ?

Comment (liamkirky) [inline Fixes.json]: oh i know what it is.

File: Fixes.json [hunk 3]
```diff
@@ -349305,6 +349310,11 @@
     "Longitude": -107.356667
   },
   {
+   "Latitude":  34.3174528,
```
Comment (Adrien-Z) [inline Fixes.json]: Remove the unnecessary space here...

File: Fixes.json [hunk 4]
```diff
@@ -468670,6 +468680,11 @@
     "Longitude": -74.76278
   },
   {
+   "Latitude":  34.3174528,
```
Comment (Adrien-Z) [inline Fixes.json]: ...and here.

**Comment by liamkirky** [comment]
Can someone merge this for me?

**Comment by indianbhaji** [comment]
@liamkirky You need to resolve the conflicts which either you need to do or a contributor needs to do

**Comment by liamkirky** [comment]
i already have changed the mistakes.

**Comment by liamkirky** [comment]
I have reviewed it and corrected it what shall i do now?

**Comment by liamkirky** [comment]
going to use this one

**Comment by Adrien-Z** [comment]
Well, you won't be able to add any commits to this anyways... It is already merged. 
Please create a new pull request. Thank you! :)

**Comment by dushyants19** [comment]
https://goo.gl/SQzPei
Hello World · GitHub Guides


https://goo.gl/pyJ0wW
About pull requests - User Documentation - GitHub Help

**Pull request merged**